core/src/validator.rs

```diff
@@ -527,12 +527,14 @@ impl ValidatorTpuConfig {
     pub fn new_for_tests(tpu_enable_udp: bool) -> Self {
         let tpu_quic_server_config = QuicServerParams {
             max_connections_per_ipaddr_per_min: 32,
+            coalesce_channel_size: 100_000, // smaller channel size for faster test
```

Comment:
Any idea what causes the tests fail with bigger channels?
I am not sure why should a bounded  channel (with bigger capacity) make the tests slower or fail?

Reply:
I tracked the call to 

    let (sender, receiver) = async_bounded(coalesce_channel_size);
Which can take 1 second to do the allocation:


I am guessing it is probably spent time allocating the buffer:

https://github.com/smol-rs/concurrent-queue/blob/master/src/bounded.rs#L62